clyde.sh

```diff
@@ -2,14 +2,37 @@
 shopt -s extglob
 
 if [ ! -d .venv ]; then
+  echo ""
   echo "Configuring Clyde..."
   git clone https://github.com/ClydeReborn/API
   mv API/!(README.md) .
   rm -rf API
   python -m venv .venv
   source .venv/bin/activate
   pip install -U -r requirements.txt
-  ln -sf clyde.bot.py clyde.py
+  echo ""
+  read -p "Do you want to run a selfbot? (not recommended) [Y/n] " choice
+  case $choice in
+    Y | y)
+        echo "Making a selfbot..."
+        ln -sf clyde.selfbot.py clyde.py
+        ;;
+    *)
+        echo "Continuing to make a bot..."
+        ln -sf clyde.bot.py clyde.py
+        ;;
+  esac
+fi
+
+if [ ! -f .env ]; then
+  echo "Creating .env file..."
+  echo "TOKEN=\"<TOKEN_GOES_HERE>\"" >> .env
+  echo "OWNER=<YOUR_DISCORD_ID_GOES_HERE>" >> .env
+  echo "ERROR_CHANNEL=<ID_OF_ERROR_LOGGING_CHANNEL_GOES_HERE>" >> .env
```

Comment:
Have you heard of single-quoted strings yet?

Reply:
yeah...